src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -682,6 +713,63 @@ sub check_dither {
 }
 
 #############################################################################################
+sub large_dither_checks {
+#############################################################################################
+    # Check the subset of monitor-window-style commanding that should be used on
+    # observations with large dither.
+
+    my $self = shift;
+    my $backstop = shift;      # Reference to array of backstop commands
+    my $dither = shift;
+    my $time_tol = 10;         # Commands must be within $time_tol of expectation
+
+    # Find the associated maneuver command for this obsid.  Need this to get the
+    # exact time of the end of maneuver
+    my $manv;
+    unless ($manv = find_command($self, "MP_TARGQUAT", -1)) {
+      push @{$self->{warn}}, sprintf("$alarm Cannot find maneuver for checking dither commanding\n");
+      return;
+    }
+
+    # Now check in backstop commands for :
+    #  Dither is disabled (AODSDITH) 1 min prior to the end of the maneuver (EOM)
+    #    to the target attitude.
+    #  Dither is enabled (AOENDITH) 5 min after EOM
+    # ACA-040
+    my $t_manv = $manv->{tstop};
+    my %dt = (AODSDITH => -60, AOENDITH => 300);
+    my %cnt = map { $_ => 0 } keys %dt;
+    my $all_dither_count = 0;
+    foreach my $bs (grep { $_->{cmd} eq 'COMMAND_SW' } @{$backstop}) {
+       my %param = Ska::Parse_CM_File::parse_params($bs->{params});
+       next unless $bs->{time} >= ($t_manv - ($dt{AODSDITH} + $time_tol + 1));
+       next unless ($param{TLMSID} =~ /^AO/);
+       # Search for all relevant dither commands
+       foreach my $cmd ('AODSDITH', 'AOENDITH', 'AODITPAR') {
```

Comment:
Gotcha.  I think we have two sets of Eric notes, because you opened this issue with the "Finally, 5 min prior to the end of the observation, dither must be commanded back to standard parameters."  But I think those are basically only different in the last observation in products.  For the last observation, I just use the "processing summary" stop time for the other checks (dither, momentum dumps).